css/styles.css

```diff
@@ -0,0 +1,559 @@
+    background-color: #fff;
+    justify-content: space-between;
+    align-items: center;
+    height: 50px;
+  }
+
+  .nav-bar > a > img {
+    width: 40px;
+  }
+
+  .nav-bar-a1 {
+    margin-left: 15%;
+  }
+
+  .nav-bar > a {
+    text-decoration: none;
+  }
+
+  .red-color {
+    color: #ec5242;
+  }
+
+  .nav-bar-h56 {
+    border: 3px solid #ec5242;
+    margin-right: 15%;
+    padding: 5px;
+  }
+
+  .intro-text > h1,
+  .intro-text > h3,
+  .intro-text > p {
+    padding-left: 15%;
+  }
+
+  .intro-text > h3 {
+    padding-top: 7%;
+  }
+
+  #para-border {
+    display: block;
+    width: 40%;
+    margin-left: 15%;
+  }
+
+  .intro-text > p {
+    margin-left: 13.5%;
+  }
+
+  .intro-text {
+    height: 90vh;
+  }
+
+  .main-program-container {
+    display: flex;
+    flex-direction: row;
+    text-align: center;
+    width: 70%;
+    margin: 0 auto;
+  }
+
+  .main-program-item {
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    width: 20%;
+    margin-right: 2px;
+  }
+
+  .main-program-container > .main-program-item > img {
+    padding-top: 15px;
+    margin-right: 0;
+  }
+
+  .main-program-button {
+    display: none;
+  }
+
+  .main-program > h3 {
+    color: white;
+    text-decoration: underline;
+    margin-top: 20px;
+    padding-bottom: 50px;
+  }
+
+  .partners-mobile {
+    display: none;
+  }
+
+  .partners {
+    display: block;
+    background-color: #414246;
+  }
+
+  .partners > .partners-desktop {
+    padding-bottom: 40px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .partners > h3 {
+    display: block;
+    width: max-content;
+    margin: 0 auto;
+    color: white;
+    padding-top: 20px;
+  }
+
+  #body-hr3 {
+    height: 1px;
+    width: 3%;
+    margin: 10px auto;
+    background-color: red;
+    border: none;
+    margin-bottom: 10px;
+  }
+
+  .row > .col > img {
+    width: 100px;
+  }
+
+  #row1,
+  #row2 {
+    display: none;
+  }
+
+  #row12 {
+    margin-bottom: 20px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .col {
+    margin-right: 30px;
+  }
+
+  .footer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 70px;
+    background-color: #d3d3d3;
+    padding-bottom: 20px;
+    padding-top: 20px;
+  }
+
+  .footer > div {
+    margin-right: 30%;
+    color: black;
+  }
+
+  #footer-logo {
+    margin-left: 20%;
+    margin-right: 10%;
+    width: 50px;
+  }
+
+  .speaker-hr {
+    width: 3%;
+  }
+
+  .speaker-container {
+    width: 70%;
+    margin: 0 auto;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .speaker {
+    width: 40%;
+  }
+
+  .speaker-button {
+    display: none;
+  }
+
+  .speaker-container > .speaker > img {
+    width: 35%;
+  }
+
+  .speaker > .speaker-info > h5 {
+    font-weight: 400;
+    font-size: 10px;
+  }
+
+  .speaker > .speaker-info > p {
+    font-weight: 400;
+    font-size: 10px;
```

Comment:
- [ OPTIONAL] Your member images should have backgrounds, as specified in the previous review. You should improve this so that your project seems more professional. :wink: